torch/csrc/distributed/rpc/python_rpc_handler.cpp

```diff
@@ -63,7 +64,12 @@ py::object PythonRpcHandler::loadPythonUDFResult(
 
 py::object PythonRpcHandler::runPythonUDF(
     const SerializedPyObj& serializedObj) {
+  auto start = std::chrono::high_resolution_clock::now();
   pybind11::gil_scoped_acquire ag;
+  auto dur = std::chrono::duration_cast<std::chrono::microseconds>(
+      std::chrono::high_resolution_clock::now() - start);
+  LOG(INFO) << "runPythonUDF: Took " << dur.count() << " us to acquire GIL.";
```

Comment:
Oh I don't think we can add a LOG(INFO) here as its gonna be too too noisy. For now may be keep track of avg and then we can figure out how to expose this information.